test/util/TestPlugin.js

```diff
@@ -3,20 +3,28 @@ import Formalism from 'renew-formalism';
 
 export class TestPlugin extends Formalism.Plugin {
 
+    constructor () {
+        super();
+        this.type = 'test';
+    }
+
     getMetaModel () {
         return Formalism.Ontology.MetaModel.fromJson({
             type: 'test',
             classifiers: [
                 {
-                    type: 'classifiers_1',
+                    type: 'classifier_1',
+                    labels: [ 'test' ],
```

Comment:
For what did you need this label? It does't even exist. I would name the label something like `'label_1'`.

Reply:
It is needed because of this line https://github.com/renew-js/renew-formalism/blob/27343b993aa034304deaa3be54399d3c7ccde1fe/src/ontology/metamodel/Relation.js#L14

Reply:
An it'll probably be needed for future labelling tests. I'll rename it to `label_1`.

Reply:
It should not fail, because of missing labels.